docs/migrations.md

```diff
@@ -0,0 +1,6 @@
+#### Migrations Tracker:
+
+[1679618910864-migration](https://github.com/bcgov/PaymentCommonComponent/blob/9ba59fecf488e3779ad63a08d876f9941c8e1da3/apps/backend/src/database/migrations/1679618910864-migration.ts)
```

Comment:
I mean more like a template on what to include in each migration relating to functional changes, but this can be done later.